manifests/concourse-manifest/concourse-base.yml

```diff
@@ -2,7 +2,7 @@
 meta:
   stemcell:
     name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
-    version: "3421.20"
+    version: "3445.15"
```

Comment:
shouldn't this be 3445.16?

Reply:
Good catch. I forgot to change it after the decision to change to 16. I've rebased and am testing it in my dev environment now - will let you know when it's finished.

Reply:
It worked ✅